src/olympia/stats/utils.py

```diff
@@ -27,13 +27,25 @@ def rows_to_series(rows, filter_by=None):
             'end': row['submission_date'],
         }
         if filter_by:
-            item['data'] = {
-                d['key']: d['value'] for d in row.get(filter_by, [])
-            }
-
+            # This filter is special because we have two columns instead of
+            # one.
             # See: https://github.com/mozilla/addons-server/issues/14411
+            # See: https://github.com/mozilla/addons-server/issues/14832
             if filter_by == AMO_TO_BIGQUERY_COLUMN_MAPPING['apps']:
-                item['data'] = {FIREFOX.guid: item['data']}
+                item['data'] = {
+                    ANDROID.guid: {
+                        d['key']: d['value']
+                        for d in row.get('dau_by_fenix_build', [])
+                    },
+                    FIREFOX.guid: {
+                        d['key']: d['value']
+                        for d in row.get('dau_by_app_version', [])
```

Comment:
I'd prefer there was an implementation that doesn't end up with `dau_by_app_version` and `dau_by_fenix_build` being hardcoded both here and in the constant above 😢 But I couldn't think of anything that wasn't horribly overcomplicated.